<!--- NOTE! THIS FILE IS AUTOMATICALLY GENERATED, IF YOU ARE READING THIS, YOU ARE EDITING THE WRONG FILE --->
**usbtool**: https://github.com/jakeogh/usbtool
### Installation:

#### Debian 11

    1. sudo apt-get install python3-pip -y

    2. pip install --force-reinstall git+https://git@github.com/jakeogh/usbtool

    3. source ~/.profile

### Examples:
```
$ usbtool find-device --help
Usage: usbtool find-device [OPTIONS] COMMAND_HEX RESPONSE_HEX

Options:
  --usb-id TEXT
  --data-dir DIRECTORY  [default: /home/user/.usbtool/2025_01_30]
  --baud-rate INTEGER   [default: 9600]
  --log-serial-data
  --timeout INTEGER     [default: 1]
  --verbose
  --dict
  --verbose-inf
  --help                Show this message and exit.

$ usbtool find-device --baud-rate 921600 100253411003 065341
ic| 1738270333.442 30884 usbtool:8<click>â†’ /usbtool.py@ _find_device():241- _devices: ['/dev/ttyUSB1', '/dev/ttyUSB0']
serial_url='spy:///dev/ttyUSB1?file=/dev/null'
ic| 1738270334.576 30884 usbtool:8<click>â†’ /usbtool.py@ _find_device():256- _tx_bytes: b'\x10\x02SA\x10\x03'
ic| 1738270335.583 30884 usbtool:8<click>â†’ /usbtool.py@ _find_device():259- _bytes_read: b'', _expected_rx_bytes: b'\x06SA'
serial_url='spy:///dev/ttyUSB0?file=/dev/null'
ic| 1738270336.592 30884 usbtool:8<click>â†’ /usbtool.py@ _find_device():256- _tx_bytes: b'\x10\x02SA\x10\x03'
ic| 1738270338.597 30884 usbtool:8<click>â†’ /usbtool.py@ _find_device():259- _bytes_read: b'\x06SA', _expected_rx_bytes: b'\x06SA'
¬/dev/ttyUSB0
```
